packages/components/radio/src/radio-group.vue

```diff
@@ -65,6 +83,19 @@ const name = computed(() => {
   return props.name || radioId.value
 })
 
+const aliasProps = computed(() => ({
+  ...radioDefaultProps,
+  ...props.props,
+}))
+const getOptionProps = (option: Record<string, any>) => {
+  const base = {
+    label: option[aliasProps.value.label],
+    value: option[aliasProps.value.value],
+    disabled: option[aliasProps.value.disabled],
+  }
+  return { ...option, ...base }
+}
```

Comment:
Same remark as https://github.com/element-plus/element-plus/pull/21545/files#r2309488335.

```suggestion
  return {
    label: option[aliasProps.value.label],
    value: option[aliasProps.value.value],
    disabled: option[aliasProps.value.disabled],
  }
}
```

Reply:
> Ok that's good, we have to mention it in documentation then.
> 
> We can be inspired by the cascader component:

Thanks for the feedback 🙏. I think it’s sufficient to just update the description of options to indicate support for more attributes. As for props, customizing the three — label, value, and disabled — should be enough. What do you think?

Reply:
Ok sounds good to me, it feels a bit hazy but we'll gonna polish the documentation based on the user report.

Reply:
> Ok sounds good to me, it feels a bit hazy but we'll gonna polish the documentation based on the user report.

I agree, we can keep it consistent with the checkbox for now and refine later with user feedback. Thanks!